module/zfs/zfs_ioctl.c

```diff
@@ -5993,6 +6032,10 @@ zfs_ioctl_init(void)
 	    zfs_ioc_recv_new, zfs_secpolicy_recv_new, DATASET_NAME,
 	    POOL_CHECK_SUSPENDED | POOL_CHECK_READONLY, B_TRUE, B_TRUE);
 
+	zfs_ioctl_register("sync", ZFS_IOC_POOL_SYNC,
+	    zfs_ioc_pool_sync, zfs_secpolicy_none, POOL_NAME,
+	    POOL_CHECK_SUSPENDED | POOL_CHECK_READONLY, B_FALSE, B_TRUE);
```

Comment:
I would recommend `allow_log=B_FALSE`, especially since you're (appropriately) using `zfs_secopolicy_none`.